File: api/TITS_API.Models/Models/Ingredient.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;
using TITS_API.Models.Interfaces;

namespace TITS_API.Models.Models
{
    [Table("Ingredients")]
    public class Ingredient : TrackedEntity
    {
        public int? PubChemCID { get; set; }
        public string PolishName { get; set; }
        public string EnglishName { get; set; }
        public string MolecularFormula { get; set; }
        public string StructureImageUrl { get; set; }
        public string GHSClasificationRaportUrl { get; set; }
        public string PubChemUrl { get; set; }
        public string WikiUrl { get; set; }


        [NotMapped]
        public List<HazardStatement> HazardStatements { get; set; }

        public override string ToString()
        {
            return PubChemCID + ", " + PolishName + ", " + EnglishName + ", " + 
                MolecularFormula + ", " + StructureImageUrl + ", " +
                GHSClasificationRaportUrl + ", " + PubChemUrl + ", " + WikiUrl;
        }
    }
}

File: mobile/app/src/main/java/com/tits/tits_mobile/models/Ingredient.java
package com.tits.tits_mobile.models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import java.io.Serializable;
import java.util.ArrayList;

@JsonIgnoreProperties
public class Ingredient implements Serializable {
    int id = 0;
    int pubChemCID = 0;
    String polishName;
    String englishName;
    String molecularFormula = null;
    String strucutreImageUrl = null;
    String ghsClasificationRaportUrl = null;
    String pubChemUrl = null;
    String wikiUrl = null;

    public ArrayList<HazardStatement> getHazardStatements() {
        return hazardStatements;
    }

    ArrayList<HazardStatement> hazardStatements;

    public Ingredient(){
        super();
    }

    public Ingredient(String polishName) {
        this.polishName = polishName;
    }


    public String getPolishName() {
        return polishName;
    }

    public void setPolishName(String polishName) {
        this.polishName = polishName;
    }

    public String getEnglishName() {
        return englishName;
    }

    public void setEnglishName(String englishName) {
        this.englishName = englishName;
    }

    public String getMolecularFormula() {
        return molecularFormula;
    }

    public void setMolecularFormula(String molecularFormula) {
        this.molecularFormula = molecularFormula;
    }

    public String getStrucutreImageUrl() {
        return strucutreImageUrl;
    }

    public void setStrucutreImageUrl(String strucutreImageUrl) {
        this.strucutreImageUrl = strucutreImageUrl;
    }

    public String getPubChemUrl() {
        return pubChemUrl;
    }

    public void setPubChemUrl(String pubChemUrl) {
        this.pubChemUrl = pubChemUrl;
    }

    public String getWikiUrl() {
        return wikiUrl;
    }

    public void setWikiUrl(String wikiUrl) {
        this.wikiUrl = wikiUrl;
    }

    public void setHazardStatements(ArrayList<HazardStatement> hazardStatements) {
        this.hazardStatements = hazardStatements;
    }
}
